File: SEP3 - TIER 3 -DBS/src/tier3NetworkPackages/TwoFieldPackage.java
package tier3NetworkPackages;

public class TwoFieldPackage extends NetworkPackage
{
    private String firstField;
    private String secondField;

    public TwoFieldPackage(NetworkType type, String firstField, String secondField, long id)
    {
        super(type, id);
        this.firstField = firstField;
        this.secondField = secondField;
    }

    public String getFirstField()
    {
        return firstField;
    }

    public String getSecondField()
    {
        return secondField;
    }

    public String toString()
    {
        return getType().toString();
    }

}

File: SEP3 - TIER1/Gradebook/NetworkPackages/TwoFieldPackage.cs
namespace WebApplication.NetworkPackages
{
    public class TwoFieldPackage : NetworkPackage
    {
        private string firstField;
        private string secondField;

        public TwoFieldPackage(NetworkType type, string firstField, string secondField, long id) : base(type, id)
        {
            this.firstField = firstField;
            this.secondField = secondField;
        }

        public string GetFirstField()
        {
            return firstField;
        }

        public string GetSecondField()
        {
            return secondField;
        }

        public string toString()
        {
            return GetType().ToString();
        }
    }
}